### ISSUE TITLE
[elitetorrent-biz] Exception (elitetorrent-biz): Got redirected to another domain. Try changing the indexer URL to https://elitetorrent.app/.: Parse error (Test)

### ISSUE THREAD

**Issue by ezruneko** [Jackett/Jackett#12529]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
11-09 08:17:31 Info Starting Jackett v0.19.210 
11-09 08:17:31 Info Environment version: 5.0.11 (/app/Jackett/) 
11-09 08:17:31 Info OS version: Unix 5.4.140.1 (64bit OS) (64bit process) 
11-09 08:17:31 Info Jackett variant: CoreLinuxAmdx64 
11-09 08:17:31 Info File /etc/issue: Ubuntu 18.04.6 LTS \n \l 
11-09 08:17:31 Info Running in Docker: Yes (image build: v0.19.210-ls50) 
11-09 08:17:31 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads 
11-09 08:17:31 Info App config/log directory: /config/Jackett 
11-09 08:17:31 Info Using proxy: Disabled 
11-09 08:17:31 Info Using FlareSolverr: http://flaresolverr:8191 
11-09 08:17:31 Info Using HTTP Client: HttpWebClient2
```

### Description

1) Add EliteTorrent.biz indexer
2) Execute a manual search or click in the test button.
3) If you try to change the url to the new domain https://elitetorrent.app/ throw the same parse exception and not change the url.

### Logged Error Messages

`2021-11-09 08:21:41 	Error 	

Error on 20211109082141 for EliteTorrent.biz.txtJackett.Common.ExceptionWithConfigData: Got redirected to another domain. Try changing the indexer URL to https://elitetorrent.app/.
   at Jackett.Common.Indexers.CardigannIndexer.CheckIfLoginIsNeeded(WebResult Result, IHtmlDocument document) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 847
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1470

<!DOCTYPE html>
<html style="height:100%">
<head>
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
<title> 301 Moved Permanently
</title></head>
<body style="color: #444; margin:0;font: normal 14px/20px Arial, Helvetica, sans-serif; height:100%; background-color: #fff;">
<div style="height:auto; min-height:100%; ">     <div style="text-align: center; width:800px; margin-left: -400px; position:absolute; top: 30%; left:50%;">
        <h1 style="margin:0; font-size:150px; line-height:150px; font-weight:bold;">301</h1>
<h2 style="margin-top:20px;font-size: 30px;">Moved Permanently
</h2>
<p>The document has been permanently moved.</p>
</div></div></body></html>

2021-11-09 08:21:07 	Error 	

Jackett.Common.IndexerException: Exception (elitetorrent-biz): Got redirected to another domain. Try changing the indexer URL to https://elitetorrent.app/.
 ---> System.Exception: Parse error
 ---> Jackett.Common.ExceptionWithConfigData: Got redirected to another domain. Try changing the indexer URL to https://elitetorrent.app/.
   at Jackett.Common.Indexers.CardigannIndexer.CheckIfLoginIsNeeded(WebResult Result, IHtmlDocument document) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 847
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1470
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 782
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1623
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 402
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 414
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 756
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/vsts/work/1/s/src/Jackett.Common/Services/IndexerManagerService.cs:line 306
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 132
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 26`

**Comment by garfield69** [comment]
upgrade to v0.19.215

**Comment by ezruneko** [comment]
thank you. i can confirm it that it's solved in the v0.19.215
